v2/dtos/intervalaction.go

```diff
@@ -0,0 +1,94 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package dtos
+
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/common"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/models"
+)
+
+// IntervalAction and its properties are defined in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/support-scheduler/2.x#/IntervalAction
+type IntervalAction struct {
+	common.Versionable `json:",inline"`
+	Created            int64  `json:"created,omitempty"`
+	Modified           int64  `json:"modified,omitempty"`
+	Id                 string `json:"id,omitempty" validate:"omitempty,uuid"`
+	Name               string `json:"name" validate:"edgex-dto-none-empty-string,edgex-dto-rfc3986-unreserved-chars"`
+	IntervalName       string `json:"intervalName" validate:"edgex-dto-none-empty-string,edgex-dto-rfc3986-unreserved-chars"`
```

Comment:
Why both Name and IntervalName?

Reply:
It's interval's name which refer to the associated interval.

Reply:
Right! Thansk!